src/bun.js/api/server.zig

```diff
@@ -1011,6 +1011,15 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
                     // User called .blob(), .json(), text(), or .arrayBuffer() on the Request object
                     // but we received nothing or the connection was aborted
                     if (request_js.as(Request)) |req| {
+                        if (req.signal) |signal| {
+                            // if signal is not aborted, abort the signal
+                            if(!signal.aborted()){
```

Comment:
There are two places we need this code in, grep for the comment above

Reply:
I see fixed on https://github.com/oven-sh/bun/pull/2097/commits/483a9f85c384626b9c42740eb7a6421d79510367